lib/pbench/test/unit/server/conftest.py

```diff
@@ -407,6 +405,9 @@ def provide_metadata(attach_dataset):
             "run": {"controller": "node1.example.com"},
         },
     )
+
+    Metadata.setvalue(dataset=test, key="dashboard.contact", value="you@example.com")
```

Comment:
Should line 383 (the instantiation of `test`) have been moved down to here?

Reply:
Oh, fine...